package/MDAnalysis/transformations/rotate.py

```diff
@@ -76,31 +77,44 @@ def rotateby(angle, direction, center="geometry", pbc=None, ag=None, position=No
         or 'mass'
     pbc: bool or None, optional
         If True, all the atoms from the given AtomGroup will be moved to the unit cell
-        before calculating the center
-    position: list, optional
+        before calculating the center. Warning: Wrapping/unwrapping the trajectory or 
```

Comment:
Thins is indeed the warning. Though, it has nothing to do with the PBC option. It is a more general problem that should be discussed in the body of the docstring. You can add a "Warnings" section.

The `pbc` option is about wrapping prior to calculating the center. Setting `pcs` to `False` is not going to change the fact that the coordinates are modified in a different way than the reference frame.

Reply:
You're right. I don't know what I was thinking, writing that there :confused: